server/data/data-jmap/src/test/scala/org/apache/james/jmap/api/upload/UploadServiceContract.scala

```diff
@@ -126,6 +127,7 @@ trait UploadServiceContract {
         .block())
 
     // Exceed 100 bytes limit
+    Thread.sleep(500)
```

Comment:
Why?

Reply:
answered above: https://github.com/apache/james-project/pull/2068#discussion_r1506928950

- Before Thread sleep, we invoked `cleanupUploadIfNeeded` 10 times. I want to it completed before invoked the 11th.